recipes/dask-glm/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "dask-glm" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "5a38d17538558fe6a3457cd67eed0a90a5dff51a9eaebb496efb68fc432ed89a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools_scm
+  run:
+    - dask
+    - scikit-learn
+    - scipy
+    - multipledispatch
+
+test:
+  imports:
+    - dask_glm
+    - dask_glm.estimators
+
+about:
+  home: http://github.com/dask/dask-glm
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Generalized Linear Models in Dask'
+  description: |
+      Generalized Linear Models for parallel and
+      distributed machine learning, built using dask.
+  doc_url: http://dask-glm.readthedocs.io/en/latest/
+  dev_url: https://github.com/dask/dask-glm
+
+extra:
+  recipe-maintainers:
+    - TomAugspurger
+    - moody-marlin
```

Comment:
@moody-marlin -- are you ok being added as a co-maintainer of the `dask-glm` recipe on conda-forge?

Reply:
https://github.com/dask/dask-glm/issues/54#issuecomment-334970901 👍

Reply:
Yup, I'm ok with it @synapticarbors